ceno_emul/src/platform.rs

```diff
@@ -10,8 +10,7 @@ use crate::addr::{Addr, RegIdx};
 #[derive(Clone, Debug)]
 pub struct Platform {
     pub rom: Range<Addr>,
-    // This is an `Option` to allow `const` here.
-    pub prog_data: Option<HashSet<Addr>>,
```

Comment:
The minor problem here was that `None` and `Some(empty_hash_set)` are two different ways to express the same thing.